lib/order.rb

```diff
@@ -0,0 +1,23 @@
+class Order
+
+  attr_reader :cart, :total, :status
+
+  def initialize
+    @cart = Hash.new { |hash, key| hash[key] = 0 }
+    @total = 0
+    @status = :new
+  end
+
+  def add(dish, quantity)
+    @cart[dish] += quantity
+    @total += dish.price * quantity
+  end
+
+  def canceled
+    @status = :canceled
+  end
+
+  def payed
+    @status = :payed
+  end
```

Comment:
I really like the `add` method name, but I don't like `canceled` or `payed` — can you tell me why that is?

Reply:
Both methods are changing the attribute '@status', they represent an action and in consequence their names should start with a verb, so I guess that I should call them 'cancel' and 'pay'